lib/omnibus/health_check.rb

```diff
@@ -489,7 +489,7 @@ def health_check_otool
       bad_libs = {}
       install_name = nil
 
-      yield_shellout_results("find #{project.install_dir} -type f | egrep '\.(dylib|bundle)$' | xargs otool -L") do |line|
+      yield_shellout_results("find #{project.install_dir} -type f | egrep '\.(dylib|bundle|so)$' | xargs otool -L") do |line|
```

Comment:
should we consider `sth.so.1.2`?
```suggestion
      yield_shellout_results("find #{project.install_dir} -type f | egrep '\.(dylib|bundle|so|so\.*)$' | xargs otool -L") do |line|
```

Reply:
I'm not sure, in theory there should still be a .so without version, and in the specific case of python modules, they aren't versioned... But let's see what this changes first

I fear that this change will break the macOS healtcheck and that we won't be able to merge it right away